packages/http/src/validator/validators/__tests__/headers.spec.ts

```diff
@@ -35,24 +34,6 @@ describe('HttpHeadersValidator', () => {
 
       describe('header is present', () => {
         describe('schema is present', () => {
-          describe('deserializer not available', () => {
```

Comment:
Why this test is disabled? The behaviour has changed?

Reply:
We do not have the registry anymore where you can dynamically query for a deserialiser. They are all fixed and the Type System makes sure we're selecting only the one we effectively have.

Reply:
There is a possibility that deserializer is not found: https://github.com/stoplightio/prism/pull/1430/files#diff-ec5f393de032d67a4f08a12ecde5456cR43

What I mean is that despite the typing system it is still possible that http-spec will sneak some garbage. Our code will deal correctly with the garbage ^ but we don't have a test for such behavior. Ideally, we should just rely on http-spec doing a good job but then comes reality.

Reply:
Ok I'll add the test back, but I'll still try tore move this — if Http Spec is terrible, we'll fix it there.